runtime/adapters.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
I understand that you are following the existing convention but I did say call this adapters_http. perhaps you can do the same for middlewares too.

Reply:
For consistency, this will require further changes to method names. For example, in `middlewares.go`, the handle is named `MiddlewareHandle`. In `middlewares_tchannel.go`, the handle is named `MiddlewareTchannelHandle`. For consistency, the handle in `middlewares.go` should be renamed to `MiddlewareHttpHandle`. There are other examples like this where http is left out in the name even though it is explicit for tchannel. Should I rename all these methods as well?